scripts/pre_commit_linter.py

```diff
@@ -1085,6 +1086,109 @@ def _check_directive_scope(all_files):
     return summary_messages
 
 
+class CustomHTMLParser(HTMLParser.HTMLParser):
+    """Custom HTML parser to check indentation."""
+
+    def __init__(self, filename, failed=False):
+        HTMLParser.HTMLParser.__init__(self)
+        self.failed = failed
+        self.filename = filename
+        self.previous_tag_line_number = None
+
+    def handle_starttag(self, tag, attrs):
+        line_number, column_number = self.getpos()
+        # Check the indentation of the tag.
+        if self.previous_tag_line_number is None:
+            if column_number % 2 != 0:
+                print (
+                    '%s --> Please ensure that %s tag '
+                    'in this file on line number %s '
+                    'has an indentation which is a multiple of 2 ' % (
+                        self.filename, tag, line_number))
+                self.failed = True
+        else:
+            if column_number % 2 != 0 and (
+                    line_number != self.previous_tag_line_number):
+                print (
+                    '%s --> Please ensure that %s tag '
+                    'in this file on line number %s '
+                    'has an indentation which is a multiple of 2 ' % (
+                        self.filename, tag, line_number))
+                self.failed = True
+        self.previous_tag_line_number = line_number
+
+        # Check the indentation of the attributes of the tag.
+        indentation_of_first_attribute = (
+            column_number + len(tag) + 2)
+        starttag_text = self.get_starttag_text()
+
+        for line_num, line in enumerate(
+                str.splitlines(starttag_text)):
+            if line_num == 0:
+                continue
+
+            leading_spaces_count = len(line) - len(line.lstrip())
+            list_of_attrs = []
+
+            for attr, _ in attrs:
+                list_of_attrs.append(attr)
+
+            if not line.lstrip().startswith(tuple(list_of_attrs)):
+                continue
+            if (
+                    indentation_of_first_attribute != (
+                        leading_spaces_count)):
+                line_num_of_error = line_number + line_num
+                print (
+                    '%s --> Please ensure that the '
```

Comment:
Rephrase to:
`Attribute for <tag name> tag on line <line number> should align with the leftmost attribute on line <line number of start of tag>.`

Reply:
Done, thanks!